### PR TITLE
exlude custom routes, kubeconfig edits for Online

### PR DESCRIPTION
**Pull request by sallyom**
WIP @ahardin-rh 
@Miciah please review, thanks

### CODE REVIEW

File: cli_reference/cli_by_example_content.adoc [hunk 1]
```diff
@@ -737,9 +755,6 @@
   // Show Merged kubeconfig settings.
   $ oc config view
 
-  // Show only local kubeconfig settings
-  $ oc config view --local
```
Comment (sallyom) [inline cli_reference/cli_by_example_content.adoc]: this isn't a valid command/flag AFAIK

**Comment by ahardin-rh** [comment]
@sallyom LGTM. Is this ready to merge? Thanks again for your help!

File: cli_reference/cli_by_example_content.adoc [hunk 2]
```diff
@@ -517,6 +520,21 @@
...
+
+ifdef::openshift-online[]
+== Securely expose containers via a route
+
+====
+
+[options="nowrap"]
+----
+  // Create a secure edge-terminated route.
+  $ oc create route edge --service=frontend
```
Comment (Miciah) [inline cli_reference/cli_by_example_content.adoc]: `oc create route passthrough --service=frontend` should also work.

**Review by Miciah — COMMENTED** [review]
Might be worth adding an explicit note that custom hosts and certificates are not allowed.

File: cli_reference/cli_by_example_content.adoc (hunk 2 above)
Comment (sallyom) [inline cli_reference/cli_by_example_content.adoc]: @Miciah, like so? look good? thanks

File: cli_reference/cli_by_example_content.adoc [hunk 3]
```diff
@@ -517,6 +520,24 @@
...
+
+  // Create a secure passthrough-terminated route.
```
Comment (Miciah) [inline cli_reference/cli_by_example_content.adoc]: Should be just "passthrough", not "passthrough-terminated"... although I notice the original example says "passthrough terminated"...

Comment (sallyom) [inline cli_reference/cli_by_example_content.adoc]: fixed, and fixed original example also

**Comment by sallyom** [comment]
@ahardin-rh this should be ready for a merge now, thanks!

**Review by Miciah — APPROVED** [review]

**Comment by ahardin-rh** [comment]
Thanks!

**Comment by ahardin-rh** [comment]
No revision history needed, as these are ifdef/endif statement adjustments.

**Pull request merged**